.gitignore

```diff
@@ -38,6 +38,20 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
+# Database entities
+backend/com.tngtech.apicenter.backend.connector.database.entity.SpecificationEntity
```

Comment:
Do you know where this comes from?

Reply:
I believe this is the database created when the app is running without a relational database configured. I deleted it, restarted the app and started adding specifications, and the directory has returned. Can I keep it in the `.gitignore`?

Reply:
Yes, I think it makes sense to keep it.